src/pages/Home/Home.jsx

```diff
@@ -1,97 +1,123 @@
 import "./home.css";
 import React from "react";
+import { Nav } from "../../components/Nav/nav";
+import { Footer } from "../../components/Footer/footer";
 import { Link } from "react-router-dom";
 
-
 const Home = () => {
   return (
     <>
-      <nav className="navigation-container">
-        <div className="nav-brand">
-          <Link className="header-style" to="/">
-            NOZAMA
+      <Nav />
```

Comment:
You could add `Nav` directly in the `App.js` file, outside the Routes. No need to add it to every Screen.